resource_cloud/views.py

```diff
@@ -450,10 +452,12 @@ def patch(self, provision_id):
                     self.delete(provision_id)
             else:
                 pr.state = args['state']
+                if pr.state == 'running' and user.is_admin:
```

Comment:
Should check that this is the first time we are in 'running' state, otherwise the lifetime counter will be reset